src/main/scala/abalone/Board.scala

```diff
@@ -20,6 +20,11 @@ case class Board(
 
   def piecesOf(player: Player): PieceMap = pieces.filter(_._2.is(player))
 
+  def isEmptySquare(pos: Option[Pos]): Boolean = pos match {
```

Comment:
when you have a match with only 2 cases, a Some and None, then you can use a map or fold depending on the output.  Here I would use a fold, e.g. 
`pos.fold(false)(!this.pieces.contains(_))`

Reply:
thank you